ci/ci/templates/batch.html

```diff
@@ -7,8 +7,14 @@ <h1>Batch {{ batch['id'] }}</h1>
       {% for name, value in batch['attributes'].items() %}
       <div>{{ name }}: {{ value }}</div>
       {% endfor %}
+      <br>
+      {% for field in ['time_created', 'time_completed', 'duration'] %}
+      {% if field in batch %}
+      <div>{{ field }}: {{ batch[field] }}</div>
```

Comment:
I think this isn't quite right. I think we should have a separate block that isn't part of the attributes. Maybe give a header for attributes and then a header for the next section with the timings. Maybe call it Timings or Metadata?

If you want to combine them, then I think you need to fix the overall `{% if 'attributes' in batch %}` to not be there because we always want time_created, etc. to show up regardless of whether there are attributes.

Reply:
Good point, I've moved them under a new `Timing` heading. I still used the `attributes` css class, because it seemed like it's used in a lot of places just to get indentation, but it is perhaps a little weird.